src/util/constants.ts

```diff
@@ -103,80 +103,80 @@ export const AXIOS_ERROR_ALERT_TIME_OUT = 5000
 export const headerSearchTitle =
 [
   {
-    text: 'Routing Slip Number',
+    title: 'Routing Slip Number',
     align: 'start',
-    value: 'routingSlipNumber',
+    key: 'routingSlipNumber',
```

Comment:
v-data-table can't display headers if it doesn't has (title/value) && key.
ref: https://github.com/vuetifyjs/vuetify/blob/master/packages/vuetify/src/components/VDataTable/types.ts#L32-L41